vllm/entrypoints/openai/serving_completion.py

```diff
@@ -93,10 +93,10 @@ async def create_completion(self, request: CompletionRequest,
                 prompt_adapter_request,
             ) = self._maybe_get_adapters(request)
 
-            tokenizer = await self.engine.get_tokenizer(lora_request)
+            tokenizer = await self.vllm_backend.get_tokenizer(lora_request)
```

Comment:
Presumably the idea is that these would never actually be remote calls and instead live within the "client" layer of the remote VLLMBackend impl (get_tokenizer, get_decoding_config, is_tracing_enabled, etc).

Reply:
We we should just pull over all the state we need at the start once

Reply:
Yes exactly, no problem with that being common logic with the single-process case too.

Though get_tokenizer might need to be slighlty different, since we may need to load lora-speciic tokenizers on the fly.